security/advancedtls/advancedtls.go

```diff
@@ -16,9 +16,14 @@
  *
  */
 
-// Package advancedtls is a utility library containing functions to construct
-// credentials.TransportCredentials that can perform credential reloading and
-// custom verification check.
+// Package advancedtls is a library containing APIs for configuring grpc
```

Comment:
Nit: all packages are libraries, and all libraries contain APIs, so I think you could remove some words here to make everything more to-the-point.  How about something like:

`Package advancedtls provides gRPC transport credentials that allow easy configuration of advanced TLS features.`

Reply:
Changed, I like that